indigo_api/models.py

```diff
@@ -945,21 +945,6 @@ class Meta:
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
-    last_submitted_by_user = models.ForeignKey(User, related_name='+', null=True, on_delete=models.SET_NULL)
```

Comment:
Nice. We may need these details later for metrics, but we can add them back if necessary.